conductor-client/src/main/java/conductor/Conductor.kt

```diff
@@ -71,6 +74,18 @@ class Conductor(private val driver: Driver) : AutoCloseable {
         }
     }
 
+    private fun waitUntilVisible(element: UiElement) {
+        repeat(10) {
```

Comment:
nit : could be a const

Reply:
It would be a bit tricky to name that constant as it is used in combination with 1000ms in `Thread.sleep`, so I would rather not do it in this specific case